src/components/interaction-designer/flow-editors/FlowEditor.vue

```diff
@@ -1,48 +1,57 @@
 <template>
```

Comment:
I'm finding the edit flow modal is no longer showing errprs when I make the flow invalid - remove the name, languages etc.